CRM/PivotData/AbstractData.php

```diff
@@ -364,7 +364,6 @@ protected function getPaginatedResults(array $apiParams, $offset = 0, $multiValu
     $apiParams['options']['offset'] = $offset;
     $entities = civicrm_api3($this->apiEntityName, 'get', $apiParams);
     $formattedEntities = $this->formatResult($entities['values']);
-    $indexes = [];
```

Comment:
This issue is probably introduced by one of the fixes in this PR: https://github.com/compucorp/uk.co.compucorp.civicrm.pivotreport/pull/126. Then there was a scenario whereby the dates are not ordered in a definite order for a particular report and there was issues of non related data ending up with a date index not related to that data. This would not have been an issue if the API request had ordered the fetched results.
 I think the fix should be okay as all the reports data are sorting results returned in a specified order.